components/asset/index.css

```
@@ -11,40 +11,41 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Asset {
-  inline-size: 100%;
-  block-size: 100%;
   display: flex;
   align-items: center;
   justify-content: center;
+
+  inline-size: 100%;
+  block-size: 100%;
 }
 
 .spectrum-Asset-image {
   max-inline-size: 100%;
   max-block-size: 100%;
   object-fit: contain;
-  transition: opacity var(--spectrum-global-animation-duration-100);
+  transition: opacity 130ms;
```

```suggestion
  transition: opacity var(--spectrum-animation-duration-100);
```

[We defined these in our own tokens package](https://github.com/adobe/spectrum-css/blob/5f365eb1f03d699ac81db6469711b58d5d79d823/tokens/custom-spectrum/custom-vars.css#L21).